requirements.txt

```diff
@@ -3,7 +3,7 @@ canonicalwebteam.discourse==2.0.3
 canonicalwebteam.flask-base==0.6.4
 canonicalwebteam.search==0.2.1
 canonicalwebteam.image-template==1.3.1
-canonicalwebteam.store-api==2.3.7
+git+git://github.com/jkfran/canonicalwebteam.store-api@package-metadata#egg=canonicalwebteam.store-api==2.3.8
```

Comment:
Waiting on https://github.com/canonical-web-and-design/canonicalwebteam.store-api/pull/49